lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -923,6 +923,12 @@ bool LocalEnforcer::update_tunnel_ids(
     schedule_revalidation(*session, csr.revalidation_time(), nullptr);
   }
 
+  if (!csr.online()) {
+    MLOG(MDEBUG) << "Credits for " << imsi << " and session "
+                 << session->get_session_id() << "will not be installed because"
+                 << " Online is 0 on CCR-A (AVP 1009) ";
```

Comment:
@uri200 
This may have a bad side effect of the corresponding policies not being installed. (I guess it's not a side effect if it's intentional. In that case can we just add a comment indicating this?)
 If I remember correctly, we look at `charging_credits_received` to check whether to install the rules or not.